packages-ts/gauntlet-terra-contracts/src/commands/contracts/migration/index.ts

```diff
@@ -0,0 +1,84 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { TerraCommand, TransactionResponse } from '@chainlink/gauntlet-terra'
+import { AccAddress, MsgMigrateContract } from '@terra-money/terra.js'
+import { CONTRACT_LIST } from '../../../lib/contracts'
+
+type CommandInput<MigrationMessage> = {
+  contract: string
+  newCodeId: number
+  migrationMsg: MigrationMessage
+}
+
+export type MigrationMessageMaker<MigrationMessage> = (flags, args) => MigrationMessage
+
+export const makeMigrationCommand = <MigrationMessage>(
+  contract: CONTRACT_LIST,
+  makeMigrationMessage: MigrationMessageMaker<MigrationMessage>,
+) => {
+  return class MigrateContract extends TerraCommand {
```

Comment:
This command has nothing to do with our contracts. We should build functionality to migrate any contract, not just Chainlink contracts in this package.

This should be moved to `gauntlet-terra` base package, so either try to do that now or add an inline comment noting the next steps oh how to restructure this.